From ba0ebca5560233fdc38d585d21f4b09a323b86ca Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Thu, 24 Jul 2014 20:07:48 -0400
Subject: [x86] quirks: Mark Cave Creek chipset device id as supported

Message-id: <20140724200747.24986.30475.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 86643
O-Subject: [PATCH RHEL7.1 BZ1087659] Mark Cave Creek chipset device id as supported
Bugzilla: 1087659
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1087659

Description:
------------
Remove the warning message for Intel Corporation 8900 Series Chipset not being supported.

RHEL Version Found:
-------------------
RHEL 7.0

Brew:
-----
https://brewweb.devel.redhat.com/taskinfo?taskID=7732797

Upstream:
---------
This is a RHEL only change.

Test Status:
------------
Tested by me using intel-shumway-01 which has the 8900 Series Chipset.

before the patch
[   16.582609] Warning: Intel Corporation 8900 Series Chipset - this hardware has not undergone testing by Red Hat
and might not be certified. Please consult https://hardware.redhat.com for certified hardware.

after the patch
the above warning msg is no longer written out.

---------------------------------------------------------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/quirks.c b/arch/x86/kernel/quirks.c
index 7d3a589..04ee1e2 100644
--- a/arch/x86/kernel/quirks.c
+++ b/arch/x86/kernel/quirks.c
@@ -571,18 +571,3 @@ DECLARE_PCI_FIXUP_FINAL(PCI_VENDOR_ID_AMD, PCI_DEVICE_ID_AMD_15H_NB_F5,
    quirk_amd_nb_node);
 
 #endif
-/*
- * RHEL7.0 does not support Intel Corporation 8900 Series Chipset.  Use
- * the Gigabit Connection to accomplish this.
- */
-static void quirk_intel_DH8900CC(struct pci_dev *dev)
-{
- static int printed;
-
- /* Some platforms have many of these devices.  Print warning once. */
- if (!printed) {
-  mark_hardware_unsupported("Intel Corporation 8900 Series Chipset");
-  printed = 1;
- }
-}
-DECLARE_PCI_FIXUP_FINAL(PCI_VENDOR_ID_INTEL, 0x2310, quirk_intel_DH8900CC);
-- 
1.7.1